server/settings/publishers.py

```diff
@@ -1091,7 +1087,9 @@ class PublishersModel(BaseSettingsModel):
         "enabled": False
     },
     "CollectFbxCamera": {
-        "enabled": False
+        "enabled": False,
+        "optional": True,
```

Comment:
Previously it was not optional - should we preserve that behavior so that it's by default still not optional unless studio enables it? :)

Reply:
yeah, I do agree with you, that we should keep it as ```optional``` ```false``` as defaults (so the TDs can expose it if desired)